dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/OrganisationUnitObjectBundleHook.java

```diff
@@ -104,6 +105,11 @@ public void postCommit( ObjectBundle bundle )
                 .getOpeningDate()) );
         }
 
+        if ( organisationUnit.getCoordinates() != null && !MathUtils.isCoordinate( organisationUnit.getCoordinates() ) )
```

Comment:
How will this work if featureType == POLYGON?

Reply:
or "MULTIPOLYGON"?  

I think you are going to have to use JTS or someother means to actually validate the geometry is valid, which may be non-trivial. I wonder if its worth beginning to validate only geometries which claim to be coordinates, and then maybe leave the validation of polygons/multipolygons as another feature, which might be more work to implement?